google-cloud-datastore/src/main/java/com/google/cloud/datastore/DatastoreImpl.java

```diff
@@ -103,13 +105,18 @@ static class ReadWriteTransactionCallable<T> implements Callable<T> {
     private final TransactionCallable<T> callable;
     private volatile TransactionOptions options;
     private volatile Transaction transaction;
+    @Nonnull private final Context transactionTraceContext;
 
     ReadWriteTransactionCallable(
-        Datastore datastore, TransactionCallable<T> callable, TransactionOptions options) {
+        Datastore datastore,
+        TransactionCallable<T> callable,
+        TransactionOptions options,
+        @Nonnull Context parentTraceContext) {
       this.datastore = datastore;
       this.callable = callable;
       this.options = options;
       this.transaction = null;
+      this.transactionTraceContext = parentTraceContext;
```

Comment:
nit - can we keep the variables and parameters all the same name?

Reply:
Done.